### ISSUE TITLE
[mangos-tbc] Guard Ranged Weapons

### ISSUE THREAD

**Issue by LewisPotato** [cmangos/issues#1857]
**Current behavior**: 
Guards do not use they're ranged weapons when needed.
**Expected behavior**: 
Guards should use they're ranged weapons when needed for ranged attacks.
**Steps to reproduce**:

1. Find a guard such as a Stormwind Guard or Deathguard Elite.
2. And there you have it. :-)
3.

**Client version**: 
2.4.3.8606
**Commit hash**: 
CMaNGOS TBC/0.18 (2019-01-28T18:59:18+01:00 - 2dad851707f224c4dab13ca1ffafc13dfa87b37e) for Win64 (little-endian) [world-daemon]
**Database version**: 
TBC-DB 1.7.0 'A New Hope' for CMaNGOS-TBC
**Operating system**: 
Windows Server 2016: Standard

**other**:
I'll be posting a list of affected guards.

[//]: # (If this is a crash report, include the crashlog from a debug build with https://gist.github.com/)

**Comment by AnonXS** [comment]
proof?

**Comment by Grz3s** [comment]
if you asking about entry 68
```code
Detected build: V3_3_0a_11159
[2] UpdateType: CreateObject1
[2] GUID: Full: 0xF1300000440C8F5B Type: Creature Entry: 68 Low: 823131
...
[2] UNIT_FIELD_FACTIONTEMPLATE: 11/1.541428E-44
[2] UNIT_VIRTUAL_ITEM_SLOT_ID: 1899/2.661066E-42
[2] UNIT_VIRTUAL_ITEM_SLOT_ID + 1: 143/2.003857E-43
[2] UNIT_FIELD_FLAGS: 32768/4.591775E-41
```
its not and 4.x.x so no flying in azeroth...

**Comment by LewisPotato** [comment]
> if you asking about entry 68
> 
> ```
> Detected build: V3_3_0a_11159
> [2] UpdateType: CreateObject1
> [2] GUID: Full: 0xF1300000440C8F5B Type: Creature Entry: 68 Low: 823131
> ...
> [2] UNIT_FIELD_FACTIONTEMPLATE: 11/1.541428E-44
> [2] UNIT_VIRTUAL_ITEM_SLOT_ID: 1899/2.661066E-42
> [2] UNIT_VIRTUAL_ITEM_SLOT_ID + 1: 143/2.003857E-43
> [2] UNIT_FIELD_FLAGS: 32768/4.591775E-41
> ```
> its not 4.x.x so no flying in azeroth...

Sorry, but what?

**Comment by Grz3s** [comment]
this is sniffed info from 3.3.0 build and it says that https://www.wowhead.com/npc=68/stormwind-city-guard
has only 2 weapons:
```code
[2] UNIT_VIRTUAL_ITEM_SLOT_ID: 1899/2.661066E-42
[2] UNIT_VIRTUAL_ITEM_SLOT_ID + 1: 143/2.003857E-43
```
no range waepon .. your info comes or from diff guards or .. from post 4.x.x

**Comment by LewisPotato** [comment]
> this is sniffed info from 3.3.0 build and it says that https://www.wowhead.com/npc=68/stormwind-city-guard
> has only 2 weapons:
> 
> ```
> [2] UNIT_VIRTUAL_ITEM_SLOT_ID: 1899/2.661066E-42
> [2] UNIT_VIRTUAL_ITEM_SLOT_ID + 1: 143/2.003857E-43
> ```
> no range weapon .. your info comes or from diff guards or .. from post 4.x.x

No, this comes from the cmangos world DB for TBC. Going to open a ticket with Blizzard (highly likely to find out that way) but will give it a go.

**Comment by Tobschinski** [comment]
The original commit, when all of this equipment was added, was done in 2014 (over at TC). 
(68, 1899, 143, 2551)
This could have been incorrectly used in all cmangos dbs.

**Comment by Ve1vet** [comment]
@LewisPotato from your liknk https://www.wowhead.com/npc=68/stormwind-city-guard#comments:id=223433
All set in comments ;)
